src/clj/puppetlabs/puppetserver/jruby_request.clj

```diff
@@ -0,0 +1,71 @@
+(ns puppetlabs.puppetserver.jruby-request
```

Comment:
I should have done this in a separate commit for better reviewability, sorry, but all of the functionality here was just moved over from the `request-handler-core` namespace.  I did that so that the environment classes handling in `master-core` could use this same functionality, keeping the basic request handling functionality simliar to that of other Puppet Ruby-bound HTTP requests.